modVM/src/org/aion/vm/ExecutionContext.java

```diff
@@ -31,277 +31,254 @@
      */
     public DataWord nrgPrice() {
         return nrgPrice;
     }
 
     /**
-     * Returns the nrg limit in current environment.
-     *
-     * @return
+     * @return the nrg limit in current environment.
      */
     public long nrgLimit() {
         return nrgLimit;
     }
 
     /**
-     * Returns the deposited value by instruction/transaction.
-     *
-     * @return
+     * @return the deposited value by instruction/trannsaction.
      */
     public DataWord callValue() {
         return callValue;
     }
 
     /**
-     * Returns the call data.
-     *
-     * @return
+     * @return the call data.
      */
     public byte[] callData() {
         return callData;
     }
 
     /**
-     * Returns the execution depth.
-     *
-     * @return
+     * @return the execution stack depth.
      */
     public int depth() {
         return depth;
     }
 
     /**
-     * Returns the call kind.
-     *
-     * @return
+     * @return the transaction kind.
      */
     public int kind() {
         return kind;
     }
 
     /**
-     * Returns the flags.
-     *
-     * @return
+     * @return the transaction flags.
      */
     public int flags() {
         return flags;
     }
 
-    // =============================
-    // Block context
-    // =============================
-
     /**
-     * Returns the block's beneficiary address.
-     *
-     * @return
+     * @return the block's beneficiary.
      */
     public Address blockCoinbase() {
         return blockCoinbase;
     }
 
     /**
-     * Returns the block number.
-     *
-     * @return
+     * @return the block number.
      */
     public long blockNumber() {
         return blockNumber;
     }
 
     /**
-     * Returns the block timestamp.
-     *
-     * @return
+     * @return the block timestamp.
      */
     public long blockTimestamp() {
         return blockTimestamp;
     }
 
     /**
-     * Returns the block nrg limit.
-     *
-     * @return
+     * @return the block energy limit.
      */
     public long blockNrgLimit() {
         return blockNrgLimit;
     }
 
     /**
-     * Returns the block difficulty.
-     *
-     * @return
+     * @return the block difficulty.
      */
     public DataWord blockDifficulty() {
         return blockDifficulty;
     }
 
-    // =============================
-    // Extra info
-    // =============================
-
     /**
-     * Returns the transaction helper.
-     *
-     * @return
+     * @return the transaction helper.
      */
     public ExecutionHelper helper() {
         return helper;
     }
+
+    /**
+     * Sets the transaction address to address.
+     *
+     * @param address The new address.
+     */
+    public void setAddress(Address address) {
+        this.address = address;
+    }
+
+    /**
+     * Sets the transaction hash to txHash.
+     *
+     * @param txHash The new transaction hash.
+     */
+    public void setTransactionHash(byte[] txHash) { this.txHash = txHash; }
```

Comment:
newline

Reply:
what's the issue here?

Reply:
It looks awkward to not have a newline between the left curly brace { and the method body; and between right curly brace } and the method body.

All the other methods are formatted that way; i.e. setAddress, getEncodingLength.

Reply:
sorry intellij tucks the one-line methods like that and I couldn't see the difference